### ISSUE TITLE
Custom highlighting for decompiler

### ISSUE THREAD

**Issue by 00rsiere** [NationalSecurityAgency/ghidra#785]
**Is your feature request related to a problem? Please describe.**
When analyzing functions using the decompiler, it can be difficult to follow the usage of specific variables. For example, if the user is trying to analyze relations between specific variables, it can be hard to follow the locations in which these variables are used in the code. The highlight feature of the decompiler (middle mouse button) can be leveraged to alleviate this issue. But, as far as I can see, it is only possible to highlight one variable at a time.

**Describe the solution you'd like**
A feature in the decompiler that enables the user to highlight different tokens in the decompiled code. This feature can help the user keep track of variable usage in functions and allow better and faster analysis.

**Describe alternatives you've considered**
#784 

**Additional context**
![image](https://user-images.githubusercontent.com/52822209/61163273-cb95d000-a515-11e9-908b-0d60505c23bb.png)

**Comment by saruman9** [comment]
I think, that it may be closed by https://github.com/NationalSecurityAgency/ghidra/commit/59e88c6ca943d5a1d58b8cac18ec1496f8552340.

**Comment by ryanmkurtz** [comment]
> I think, that it may be closed by [59e88c6](https://github.com/NationalSecurityAgency/ghidra/commit/59e88c6ca943d5a1d58b8cac18ec1496f8552340).

I think you are correct.
